.github/workflows/R-CMD-check-dev.yaml

```diff
@@ -17,7 +17,6 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {os: ubuntu-latest, r: '4.0.5', repos: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
           - {os: ubuntu-latest, r: '4.1.3', repos: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
```

Comment:
should we also include latest r release in dev checks since its required in main, to avoid surprises when pushing releases from dev to main?

Reply:
Yes, absolutely! I didn't want to change EVERYTHING all at once, but I'll go ahead and do that!